service/terraform-plugin/src/main/java/com/solace/maas/ep/event/management/agent/plugin/terraform/configuration/TerraformProperties.java

```diff
@@ -7,6 +7,6 @@
 @Service
 @Data
 public class TerraformProperties {
-    @Value("${COMMAND_PATH:${user.home}${file.separator}tfcommands}")
+    @Value("${app.commandroot:${user.home}}${file.separator}tfcommands")
```

Comment:
Do we need to update README instructions to include this new variable for those running EMA locally as a docker container?

Reply:
This issue seems to be specific to the way we are deploying the container in k8s. I'm not sure if it's a rootless deployment or something, but this issue does not occur when running the docker container using the commands in the README.  If you don't specify the `app.commandroot` Spring variable, it defaults to the previous behaviour of using the `user.home` as the root for the `command` and `tfcommand` directories so a user following instructions from our README will see no change.
If you'd like, I can add a section explaining how the new `app.commandroot` spring variable works and how they can use it to customize where the logs and terraform artifacts are created. Let me know and I can add it.

Reply:
I see, ok let's leave things this level of flexibility is not needed for local users. If some asks for it we can update the readme of how to do it.